server/routes/trees/trees-router.js

```diff
@@ -46,10 +46,12 @@ treesRouter.post('/', async (req, res) => {
 
   // creates id here when new trees are planted in the ground
   // haven't gone to vector tiles yet.
-  const id = req.body.id ? req.body.id : createIdForTree(req.body);
+  const id = req?.body?.id ? req.body.id : createIdForTree(req.body);
```

Comment:
Nit: Couldn't we do `req?.body?.id ?? createIdForTree(req.body);`?

Otherwise, looks good to me!